### DISCUSSION TITLE
42. 왜 TCP/IP는 더 단순화되었는지, 그리고 이로 인한 장단점은 무엇인지 설명해보세요.

### DISCUSSION THREAD

**Discussion by linirini** [jewelry-box/cs-interview#43] [General]
.

**Comment by eun-byeol** [comment]
실제 네트워크 통신을 구현하기 위한 `실용 중심 설계` 설계와 `초기 인터넷 환경`(ARPANET* 등)에 맞추어 단순화되었습니다.
장점으로는 단순한 계층 구조로 `구현과 사용이 쉽고`, 중복된 기능을 없애 `불필요한 오버헤드를 줄일` 수 있다는 점이 있고,
단점으로는 단순화로 인해 계층 간 역할 분리가 명확하지 않아 `특정 계층에서 다양한 작업을 처리`하는 경우도 발생한다는 점이 있습니다.

---
> [ARPANET](https://developer.mozilla.org/ko/docs/Glossary/Arpanet)

1969년에 민감한 군사 데이터를 전송하고 미국 전역의 주요 연구 그룹을 연결하기 위한 초기 네트워크
